src/tasks/task1.js

```diff
@@ -0,0 +1,28 @@
+let _chunk;
+const _reverse = str => {
+  return str.split('').reverse((a, b) => {
+    if (a > b) return 0;
+    if (a < b) return -1;
+  }).join(',').replace(/,/g, '')
+}
+
+export class Task1 {
+
+  constructor() {
+    this.init();
+  }
+
+  init() {
+    process.stdin.setEncoding('utf8');
+    process.stdin.on('readable', () => {
+      while ((_chunk = process.stdin.read()) !== null) {
+        process.stdout.write(`Chunk data is: ${_reverse(_chunk)}`);
```

Comment:
Could you make the next input in newline?

Reply:
Done.